app/services/ridb_service.rb

```diff
@@ -14,7 +14,7 @@ def self.get_activities
     JSON.parse(response.body, symbolize_names: true)
   end
 
-  def self.get_rec_area(latitude, longitude, radius = 50.00, activities)
```

Comment:
We had the radius default to 50 miles if the user didn't specify in the search.  We can change this back if thats not something you all decided upon.